tests/src/Functional/ReorderChildrenContentModerationFunctionalTest.php

```diff
@@ -97,8 +95,8 @@ public function testReorderingForDraftParent(): void {
       'Child 3',
       'Child 2',
       'Child 1',
-    ]), array_map(function (Node $node) {
-      return $node->getId();
+    ]), array_map(function (\StdClass $node) {
+      return $node->id;
```

Comment:
not a fan of \StdClass, can we use fetchObject in the queries and have a value object instead?